### DISCUSSION TITLE
HOW TO SAVE THE RESULT IN PNG AND GIF?

### DISCUSSION THREAD

**Discussion by tsudamasaki** [facebookresearch/pytorch3d#997] [Q&A]
I'D LIKE TO SAVE THE DEFORMING PROCESS IN PNG AND GIF.
HOW CAN I DO THAT?11

**Comment by toyosatomimi98** [comment]
use plt.imsave()?

**Comment by patricklabatut** [comment]
Save the output frames of the deformation as PNG images and generate an animated GIF from these images with some tool (e.g. ffmpeg)?
